docs/spec.md

```diff
@@ -1420,6 +1479,10 @@ value and produces a `result` tensor. More formally, `result[i0, ..., iR-1]` =
 where `min_val = rank(min) == 0 ? min : min[i0, ..., iR-1]`,
 `max_val = rank(max) == 0 ? max : max[i0, ..., iR-1]`.
 
+Imposing an ordering on complex numbers involves surprising semantics,
+so in the future we are planning to remove support for complex numbers
+for this operation ([#560](https://github.com/openxla/stablehlo/issues/560)).
```

Comment:
not in forntend constract.

Reply:
Same as [above](https://github.com/openxla/stablehlo/pull/1076#discussion_r1099053266).